hack/test-integration.sh

```diff
@@ -33,7 +33,16 @@ if [ "$COVER" -eq 1 ]; then
 fi
 
 # Run the tests and collate the results.
-bats -t $ROOT/test/*.bats
+TESTS=""
+for item in ${BATS[@]};
```

Comment:
Just to clarify. If we make them arguments, then we must split the string in the Makefile, having the same issue there. Or am I missing something?